JPATask/src/main/java/com/instinctools/egor/mentoring/JPA/CommandLineMenu/ClientWorkingMenu.java

```diff
@@ -0,0 +1,104 @@
+package com.instinctools.egor.mentoring.JPA.CommandLineMenu;
+
+import com.instinctools.egor.mentoring.JPA.Entity.Account;
+import com.instinctools.egor.mentoring.JPA.Entity.Client;
+import com.instinctools.egor.mentoring.JPA.Service.ClientService;
+import com.instinctools.egor.mentoring.JPA.Service.DAOServiceImpl.ClientServiceImpl;
+
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Scanner;
+
+public class ClientWorkingMenu {
+
+    private ClientService clientService;
+    private Scanner scanner;
+
+    public ClientWorkingMenu(){
+        clientService = new ClientServiceImpl();
+    }
+    public void start(){
+        scanner = new Scanner(System.in);
+        int chose;
+        String login, password;
+        do{
+            System.out.println("Menu: \n" +
+                    "1 - create client\n" +
+                    "2 - work with existing client\n" +
+                    "3 - show list of all accounts\n" +
+                    "4 - delete client\n" +
+                    "0 - exit");
+            chose = Integer.parseInt(scanner.next());
+            switch(chose){
+                case 1: {
+                    System.out.println("input username: \n");
+                    login = scanner.next();
+                    System.out.println("input password: \n");
+                    password = scanner.next();
+                    try {
+                        clientService.createClient(new Client(login, password));
+                    } catch (SQLException e) {
+                        e.printStackTrace();
+                    }
+                    break;
+                }
+                case 2: {
+                    Client mainClient = showClients();
+                    if(mainClient == null)break;
+                    System.out.println("Your client: "+ mainClient.toString());
+                    AccountWorkingMenu accountMenu = new AccountWorkingMenu(mainClient);
```

Comment:
Bad solution here to create service based on specific client - services (and different view controllers also) should be stateless